src/encoded/tests/test_access_key.py

```diff
@@ -72,8 +72,8 @@ def test_access_key_principals(anontestapp, execute_counter, access_key, submitt
     ]
 
 
-def test_access_key_self_create(anontestapp, access_key, submitter):
-    extra_environ = {'REMOTE_USER': str(submitter['email'])}
+def test_access_key_self_create(anontestapp, admin):
```

Comment:
Shouldn't a submitter still have permission to create an access_key? Why did you change it to admin?